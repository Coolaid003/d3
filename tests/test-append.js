require("./../lib/env-js/envjs/node");
require("./../lib/sizzle/sizzle");
require("./../d3");

d3.select("body").append("svg:svg");

console.log("append svg:svg:");
console.log("  ", document.body.innerHTML);
console.log("");

d3.select("body").append("div");

console.log("append div:");
console.log("  ", document.body.innerHTML);
console.log("");

d3.select("body").append("#text").text("hello");

console.log("append text \"hello\":");
console.log("  ", document.body.innerHTML);
console.log("");

d3.select("body").append("#text").text(function() { return "there"; });

console.log("append text \"there\" via a function:");
console.log("  ", document.body.innerHTML);
console.log("");
