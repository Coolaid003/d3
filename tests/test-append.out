append svg:svg:
   <svg/>

append div:
   <svg/><div/>

append text "hello":
   <svg/><div/>hello

append text "there" via a function:
   <svg/><div/>hello there

